src/interdiff.c

```diff
@@ -58,6 +59,27 @@
 #define PATCH "patch"
 #endif
 
+#define COLOR_RESET     "\033[0m"
```

Comment:
`COLOR_RESET` is only used in one place and will always contain the same value, so this macro is unneeded. Remove it.